drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -63,7 +75,7 @@ class DragAndDropBlock(
         display_name=_("Title"),
         help=_("The title of the drag and drop problem. The title is displayed to learners."),
         scope=Scope.settings,
-        default=_("Drag and Drop"),
+        default=_("Drag & Drop"),
```

Comment:
in my devstack, the "&" cause the "Drag and Drop" button not show up in the "Add New Compontent" -> "Quiz" list

Reply:
![image](https://github.com/Learningtribes/xblock-drag-and-drop-v2/assets/26813045/beb77ffc-f721-4d7a-81a9-fec8de447c0b)

 - Pls git pull latest source code of `platform`. This issue has been fixed.